plugins/faucet/object.go

```diff
@@ -8,39 +8,89 @@ import (
 	// Only want to use init
 	_ "golang.org/x/crypto/blake2b"
 
+	"github.com/iotaledger/goshimmer/dapps/valuetransfers/packages/address"
 	"github.com/iotaledger/goshimmer/packages/pow"
 	"github.com/iotaledger/goshimmer/packages/tangle"
 	"github.com/iotaledger/hive.go/marshalutil"
 	"github.com/iotaledger/hive.go/stringify"
-
-	"github.com/iotaledger/goshimmer/dapps/valuetransfers/packages/address"
 )
 
 const (
-	// ObjectName defines the name of the facuet object.
+	// ObjectName defines the name of the facuet object (payload).
 	ObjectName = "faucet"
 )
 
-// Payload represents a request which contains an address for the faucet to send funds to.
-type Payload struct {
+// region Object implementation ///////////////////////////////////////////////////////////////////////////////////////
+
+// Type represents the identifier for the faucet Object type.
+var Type = tangle.PayloadType(2)
+
+// Object represents a faucet request which contains an address for the faucet to send funds to.
+type Object struct {
```

Comment:
```suggestion
type FaucetRequest struct {
```

Reply:
We have discussed and decided to use domain specific names for the model in the package (e.g., transaction, facuetRequest). Please change this accordingly 👍

Reply:
I named it as `Request`, because linter was complaining about it starting with the same name as the package. From outside, it will be `faucet.Request`.